src/components/UserLogin/UserLogin.stories.tsx

```diff
@@ -0,0 +1,13 @@
+import { type Meta, type StoryObj } from '@storybook/react';
+import UserLogin from './UserLogin.component';
+
+const meta: Meta<typeof UserLogin> = {
+  title: 'Components/UserLogin',
+  component: UserLogin,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const EmptyList: Story = {};
```

Comment:
@RandilaP You need to write a sample story for this.

Reply:
@anjula-sack Can you give me an example